build.gradle

```diff
@@ -38,24 +45,17 @@ def depVersions = [
 
 ]
 
-allprojects {
-    group = 'com.commercetools'
-    version = '2.2.0-dev' // this version is used in shadowJar.manifest.Implementation-Version and then at runtime as getImplementationVersion()
-
-    tasks.withType(JavaCompile) {
-        options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
-    }
-}
-
 subprojects {
-
     apply plugin: 'java'
 
+    group = 'com.commercetools'
+    version = '2.2.0-dev' // this version is used in shadowJar.manifest.Implementation-Version and then at runtime as getImplementationVersion()
```

Comment:
why do you have it 2.2.0 hardcoded? Do you change it everytime you release manually?

Reply:
yes. it is a development version to be visible during staging and testing. I change it every time i start a new version development.